pyccel/parser/semantic.py

```diff
@@ -1,6 +1,9 @@
 # -*- coding: utf-8 -*-
-# pylint: disable=R0201
-# pylint: disable=missing-function-docstring
+#------------------------------------------------------------------------------------------#
+# This file is part of Pyccel which is released under MIT License. See the LICENSE file or #
+# go to https://github.com/pyccel/pyccel/blob/master/LICENSE for full license details.     #
+#------------------------------------------------------------------------------------------#
+# pylint: disable=missing-module-docstring, R0201, missing-function-docstring
```

Comment:
Please do not disable missing docstrings outside of the tests/ folder.

I am happy for you to leave this file without a docstring.